### DISCUSSION TITLE
Inline button doesnt show in Master dialog

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6654] [Help]
I have created an inline button which shows in a child grid when access directly from the navigtion menu, but when its access through the master dialog page it doesnt show.
Please assist

Directly accessing the child page

![image](https://user-images.githubusercontent.com/50826372/212143241-41138457-1514-44df-8f3b-d65d02dd15fd.png)

Accessing it through the master dialog page

![image](https://user-images.githubusercontent.com/50826372/212143580-c76c7a6c-f6cd-49d1-b060-c840b06ba61d.png)

**Comment by muhammedyaman** [comment]
This must be because when you enter the page ChildIndex.cshtml calls ChildGrid.ts which has your getColumns() modification. But when you enter the master dialog MasterDialog calls ChildEditor.ts. If you just modify your editor as you did at grid it will work as expected. 

Another possibility is UserPreferences. If there is a saved UserPreference in UserPreferences table for LineItemsEditor you must delete or modify them to show newly added columns.
